core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1157,7 +1165,7 @@ package object config {
 
   private[spark] val IO_COMPRESSION_CODEC =
     ConfigBuilder("spark.io.compression.codec")
-      .doc("The codec used to compress internal data such as RDD partitions, event log, " +
+      .doc("The codec used to compress internal data such as RDD partitions, " +
```

Comment:
and .. seems like the doc is still valid if `spark.eventLog.compression.codec` is not given.

Reply:
Yep. It's recovered.